pkg/util/capabilities/capabilities.go

```diff
@@ -0,0 +1,367 @@
+package capabilities
```

Comment:
Thanks for doing this Cedric, this was something I needed to get around to implementing but hadn't had time yet :-)